pkg/api/api_test.go

```diff
@@ -90,22 +90,70 @@ var _ = Describe("Api", func() {
 				Expect(actualProfileName).To(Equal("nginx"))
 			})
 		})
+	})
 
-		When("no name query is provided", func() {
+	Context("/profiles", func() {
+		When("few profiles exist", func() {
```

Comment:
would this be more accurate:
```suggestion
		When("no query string is passed to the search endpoint", func() {
```

Reply:
done 👍🏻